### ISSUE TITLE
Model always in training mode?

### ISSUE THREAD

**Issue by LivC193** [prokia/drugVQA#7]
Hi I was wondering if you can clarify something.
I see in your code that you use `BatchNorm2D` several times. However at test time you do not switch to `model.eval()`. Why is that ? 
https://github.com/prokia/drugVQA/blob/b69ae4c2c4182bf638adf2da504c76ca1bfa48f9/model.py#L20

https://github.com/prokia/drugVQA/blob/b69ae4c2c4182bf638adf2da504c76ca1bfa48f9/trainAndTest.py#L131

Furthermore, what is the reason for using `BatchNorm2D` when your batch size is 1?
